cowprotocol/tenderly-watch-tower
# refactor: remove tenderly

# Description
Tenderly Web3 Actions require a lot of workarounds, as detailed in #34. Therefore, this PR starts the work to remove Tenderly and migrate this service to completely standalone.

# Changes

- [x] Remove root level package configuration
- [x] Moved `actions` level package configuration to the root
- [x] Configured sane defaults using `cow-sdk` settings for `eslint` etc.

## How to test

This PR covers only package configuration, and **does not address compilation of code.** To test, confirm:

1. `yarn lint`
2. `yarn fmt`
3. `yarn build`

Typechain bindings should be generated, and the respective scripts should throw their relevant errors (eslint, prettier, tsc).

## Related Issues

- Fixes #36 
- Fixes #20
